system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/cells/MetricsGraphs.js

```diff
@@ -68,14 +72,21 @@ const styles = {
         paddingLeft: 5
     },
     contentGrid: {
-        height: 186
+        marginTop: 48
     },
     toolTipHead: {
         fontWeight: 600
     },
     barChart: {
         marginTop: 40,
         marginBottom: 40
+    },
+    info: {
+        color: "#999",
+        marginTop: 8,
+        marginRight: 27,
+        fontSize: 18
+
```

Comment:
unwanted new line

Reply:
Removed in https://github.com/wso2-cellery/mesh-observability/blob/master/components/global/portal/io.cellery.observability.ui/src/components/cells/metricsGraphs/index.js